media-library-client/src/app/private-container/pages/home/video-gallery/video-tiles/video-tiles.tsx

```diff
@@ -1,28 +1,29 @@
-import { Box, Grid } from '@mui/material';
+import { Button, Grid, Rating, Typography } from '@mui/material';
 import { useQuery } from '@tanstack/react-query';
 import { useEffect, useMemo, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import HttpHelper from '../../../../../classes/http-helper';
+import ObjectStatus from '../../../../../enums/object-status';
 import { updateBreadcrumbLinks } from '../../../../../functions/bread-crumb-functions';
 import Video from '../../../../../interfaces/video';
 import VideoGallery from '../../../../../interfaces/video-gallery';
 import { RoutePrivateRoot, RouteVideoGallery, RouteVideos } from '../../../../../routes/app-routes';
+import VideoPlayer from './video-player/video-player';
+import ValidatedTextField from '../../../../../custom-components/inputs/validated-text/validated-text';
+import NavItem from '../../../../../custom-components/nav-items/nav-item';
+import IconButtonWrapper from '../../../../../custom-components/icon-button-wrapper/icon-button-wrapper';
+import IconSelector from '../../../../../custom-components/icon-loader/icon-selector';
 
 export default function VideoTiles() {
 	const params = useParams();
-	const videoGalleryID = useMemo(() => params?.videoGalleryID ?? '', []);
-	const [videoGallery, setVideoGallery] = useState<VideoGallery>();
-	const [videos, setVideos] = useState<Video[]>();
-
-	const videoGalleryGetAllQuery = useQuery({
-		...HttpHelper.videoGallery.GetByID(videoGalleryID),
-		enabled: true,
-	});
-
-	const videoGetByGalleryQuery = useQuery({
-		...HttpHelper.video.GetByGallery(videoGalleryID),
-		enabled: !!videoGalleryID,
+	const videoGalleryID = useMemo(() => params?.videoGalleryID ?? '', [params?.videoGalleryID]);
+	const [videoGallery, setVideoGallery] = useState<VideoGallery | undefined>();
+	const [pageSetup] = useState({
+		page: 1,
+		pageSize: 12,

```

Comment:
We should create a feature ticket to calculate page size based on screensize and also allow user overidess